docs/TROUBLESHOOTING.md

```diff
@@ -48,6 +48,19 @@ Once the SSH bastion has been setup, you can use either method to access the Win
   ```
 * You can now RDP into the Windows node at *localhost:2020* using an RDP client
 
+## Rebooting a Windows node
```

Comment:
If we add reboot instructions, we should provide a disclaimer emphasizing responsible use and include context for specific scenarios where this option may be necessary.

Reply:
Agree, and tried to cover it with:

`"Certain operations and updates at the system level still require a traditional reboot process to ensure changes are applied correctly and securely" `

Covering all the possible specific scenarios may be hard, given that it is sort of open-ended topic based on the user setup.